example/README.md

```diff
@@ -0,0 +1,71 @@
+# # nubison-model example
+
+This is an example of how to use the `nubison-model` library to register a user model.
+
+## ## Prerequisites
+
+- mlflow server is running on `http://127.0.0.1:5000` or set `MLFLOW_TRACKING_URI` environment variable.
+
+## ## Example structure
+
+```
+example/
+├── model.ipynb
+├── requirements.txt
+└── src/
+    └── SimpleLinearModel.py
+```
+
+- `model.ipynb`: A notebook file that shows how to register a user model and test it.
+- `requirements.txt`: A file that specifies the dependencies of the model.
```

Comment:
Is there a reason for using requirements.txt instead of conda.yaml for installing packages with Conda?

Reply:
To ensure easy acceptance by everyone, the `conda.yaml` file is designed with specific preferences, while `requirements.txt` follows a standard format.
Some of the advanced features of `conda.yaml` compared to `requirements.txt` include:

- Directly specifying the Python version:
  - This enables us to use the current Python version without needing to extract it from a separate file.
  - It allows models to be stored in a registry through serialization using cloud pickle, which is sensitive to the Python version.
- Specifying package versions within a range:
  - The `nubison-model` is used for registering models for deployment. It's preferable to pin a specific version rather than using a range.